src/main/java/com/bandwidth/sdk/messaging/BandwidthMessagingException.java

```diff
@@ -0,0 +1,49 @@
+package com.bandwidth.sdk;
+
+
+public class BandwidthMessagingException extends RuntimeException {
+
+    private static final String MESSAGE_ERROR_TYPE = "message-failed";
```

Comment:
This probably isn't necessary since it isn't being used anywhere.

Reply:
fixed